python/mlx/distributed_run.py

```diff
@@ -386,15 +386,35 @@ def node_thread(rank, host, hostfile, input_queue):
         t.join()
 
 
+def get_mpi_libname():
+    try:
+        ompi_info = run(["which", "ompi_info"], check=True, capture_output=True)
```

Comment:
I'm assuming `ompi_info` is also installed with conda, but haven't verified yet. Note that if it were not, we would return `None` and the process would follow without adding the environment vars, so everything would work as before.